libbeat/docs/shared-ssl-config.asciidoc

```diff
@@ -87,25 +87,146 @@ Example module with SSL enabled:
 ----
 endif::[]
 
-[float]
-=== Configuration options
+[discrete]
+[[ssl-common-config]]
+=== Common configuration options
```

Comment:
What is "common" supposed to imply here?

Reply:
It might be worth adding an additional sentence to the paragraph (sentence) below that explains what common means?

Reply:
Ahh, I see, it's configs that are used in both client and server. I think that should be explained in the sentence below.

Reply:
It might also be nice to add a jump table above the `Common configuration options` heading that links to the common, client, and server sections of the doc